lib/serializer/BinaryDeserializer.cpp

```diff
@@ -79,13 +80,11 @@ void CLoadFile::openNextFile(const boost::filesystem::path & fname, int minimalV
 	}
 }
 
-void CLoadFile::reportState(CLogger * out)
+void CLoadFile::reportState(vstd::CLoggerBase * out)
 {
 	out->debug("CLoadFile");
 	if(!!sfile && *sfile)
-	{
-		out->debugStream() << "\tOpened " << fName << "\n\tPosition: " << sfile->tellg();
-	}
+		out->debug("\tOpened %s \tPosition: %d", fName, sfile->tellg());
```

Comment:
@alexvins as you wish. No idea of extra tab is important to make this part of log look nicer.